.github/workflows/go-mod-major.yml

```diff
@@ -0,0 +1,25 @@
+name: Go Module Major Update Check
+on:
+  schedule:
+    - cron: '30 6 * * 1' # at 06:30 every Monday morning
```

Comment:
I've used this trick [myself](https://github.com/Al2Klimov/check_rungrep/blob/master/.github/workflows/rust-update.yml) and according to GH's eMails they're auto-suspending these jobs after the last code change and an mount of time I don't remember.